### PR TITLE
Search results design tweaks (#60073)

### PR DESCRIPTION
**Pull request by vovakulikov**
Original PR https://github.com/sourcegraph/sourcegraph/pull/60073
(cherry picked from commit c32dba7b5cecfe067c4166bb78807f03ebdeec98)

Short description, this original PR contains small design tweeks around hover and loading states

## Test plan
- CI passes

### CODE REVIEW

**Comment by vovakulikov** [comment]
cc @sourcegraph/release-guild I mention you since I've created this PR manually and not sure you saw it through backport notifications

**Review by camdencheek — APPROVED** [review]

File: client/branded/src/search-ui/results/filters/components/dynamic-filter/SearchDynamicFilter.tsx [hunk 1]
```diff
@@ -132,7 +139,12 @@ export const SearchDynamicFilter: FC<SearchDynamicFilterProps> = ({
 
                 {filtersToShow.length === 0 && (
                     <small className={styles.description}>
-                        There are no {filterKind}s to show, try to use different search value
+                        <b>We couldn’t return a {filterKind} that matched your filter input.</b> Try a more expansive
```
Comment (rrhyne) [inline client/branded/src/search-ui/results/filters/components/dynamic-filter/SearchDynamicFilter.tsx:142]: @vovakulikov  can we address this here, given the need to reduce backports? 

https://github.com/sourcegraph/sourcegraph/issues/60172

Comment (vovakulikov) [reply client/branded/src/search-ui/results/filters/components/dynamic-filter/SearchDynamicFilter.tsx:142]: @rrhyne, no. This PR originally already has been merged into the main, in order to maintain a linear history of commits and don't have any differences between the 5.3 release branch and the main branch we should follow the following flow
- Make changes and create PR in the main branch
- Merge PR into the main
- Automatically (or manually) create backport PR (we're here)
- Merge backport PR

No worries I make sure that the empty state update is backported too.

Comment (rrhyne) [reply client/branded/src/search-ui/results/filters/components/dynamic-filter/SearchDynamicFilter.tsx:142]: Roger, and thanks @vovakulikov!

**Pull request merged**